lib/colorado_lottery.rb

```diff
@@ -0,0 +1,34 @@
+class ColoradoLottery
+
+  attr_reader :registered_contestants, :winners, :current_contestants
+  def initialize()
+    @registered_contestants = Hash.new { |hash, key| hash[key] = [] }
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(person_interested, game_interest)
```

Comment:
Exactly the setup we needed here, but `person_interested` is a poor naming choice, and would be better off named `contestant` to accurately represent the object being passed.